cranelift/codegen/meta/src/shared/instructions.rs

```diff
@@ -4325,6 +4325,21 @@ pub(crate) fn define(
         .operands_out(vec![a]),
     );
 
+    ig.push(
+        Inst::new(
+            "fcvt_low_from_sint",
+            r#"
+        Convert signed integer to floating point.
+
+        Each lane in `x` is interpreted as a signed integer and converted to
+        floating point.
```

Comment:
I think we should add something about rounding like in `fcvt_from_sint` and something describing what the `low` part of the name means.

Reply:
I mean, it's not like the [SIMD documentation](https://github.com/WebAssembly/simd/blob/master/proposals/simd/SIMD.md) is much more verbose... perhaps that should get fixed too. I think we need to say something about how the two lowest `i32`s in the vector (lanes 0 and 1) are the ones converted to the `f64`. From the Intel reference:

CVTDQ2PD (128-bit Legacy SSE version)
DEST[63:0] := Convert_Integer_To_Double_Precision_Floating_Point(SRC[31:0])
DEST[127:64] := Convert_Integer_To_Double_Precision_Floating_Point(SRC[63:32])
DEST[MAXVL-1:128] (unmodified)

Reply:
Yes, this is a good point. In updating the comments I just used language of half. That is because the "fcvt_low_from_sint" can also support AVX-512 where we could convert the "four" lowest i32s. That instruction doesn't exist of course in the spec, but could be a candidate for a future iteration of the spec or relaxed simd.